atlasdb-commons/src/main/java/com/palantir/common/base/ClosableIterator.java

```diff
@@ -32,13 +35,29 @@
     default void close() { }
 
     default <U> ClosableIterator<U> map(Function<T, U> mapper) {
-        return ClosableIterators.wrap(stream().map(mapper).iterator(), this);
+        return ClosableIterators.wrap(Iterators.transform(this, (com.google.common.base.Function)mapper));
```

Comment:
nit: baseline requires a space here: `...Function) mapper);`

We may not enforce baseline compliance for `atlasdb-commons`, but we might as well avoid introducing new issues.